tests/src/Context/MessageContext.php

```diff
@@ -16,6 +16,7 @@
  * @package Drupal\nexteuropa\Context
```

Comment:
@gillesdeudon If this case appears, you can use the command "git config core.fileMode **true**".
I agree with @drupol, changing the permission is not needed in this PR.